types/office-js/index.d.ts

```diff
@@ -71130,6 +71132,10 @@ declare namespace Word {
      * The insertion location types.
      *
      * [Api set: WordApi]
+     * @remarks
+     * To be used with an API call, such as `obj.insertSomething(newStuff, location);`
+     * If the location is "Before" or "After", `newStuff` will be outside of the modified `obj`.
+     * If the location is "Start" or "End", `newStuff` will be included as part of the modified `obj`.
```

Comment:
```suggestion
     * If the location is "Before" or "After", the new content will be outside of the modified object.
     * If the location is "Start" or "End", the new content will be included as part of the modified object.
```

Reply:
Thanks for the suggestion! Got an error from GitHub while trying to commit through the UI, so I added this in subsequent command line commit.